front/src/containers/admin/AddBatch.js

```diff
@@ -345,6 +360,7 @@ function mapStateToProps(state) {
       reconveneWorstRound: false,
       randomizeExpRound: false,
       dynamicTeamSize: false,
+      dynamicOptions: null,
```

Comment:
I suspect this would be safer to be instantiated with false but I'm not sure it really matters all that much. 
```suggestion
      dynamicOptions: false,
```

Perhaps reply if this doesn't make sense, and we can work out a better option.